app/controllers/incurred_incidentals_controller.rb

```diff
@@ -12,6 +12,8 @@ def new
   end
 
   def edit
+    params[:rental_id] = @incurred_incidental.rental.id
```

Comment:
A bit odd to be manually editing request parameters to then pass to the view - seems like a function which would be served more naturally by instance variables.